contracts/upgradeable_contracts/amb_erc20_to_native/HomeAMBErc20ToNative.sol

```diff
@@ -34,9 +33,8 @@ contract HomeAMBErc20ToNative is BasicAMBErc20ToNative, BlockRewardBridge, Rewar
         uint256 _requestGasLimit,
         uint256 _decimalShift,
         address _owner,
-        address _feeManager,
         address _blockReward
-    ) external onlyRelevantSender returns (bool) {
+    ) public onlyRelevantSender returns (bool) {
```

Comment:
Should it be public or internal?

Reply:
I think it should be public, so that it is possible to initialize a mediator by both: `initialize` and `rewardableInitialize`